test

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+clear
+echo "HaI"
+pwd
+find *.log>report.txt
+find *.txt>>report.txt
+tar -zcvf test.tar.gz Linux
```

Comment:
The requirement is to tar and compress only the *.txt and *.log from this folder. tar and compress should ignore all other files in this folder. Can you accomplish that?